### ISSUE TITLE
NTR: Filipin stain of fibroblasts: un-esterified cholesterol accumulation (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9212]
New term request Filipin stain of fibroblasts: un-esterified cholesterol accumulation (elthat_101212170527):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000158
- label: Unesterified cholesterol accumulation in cultured fibroblasts
- parents: [Abnormality of lysosomal metabolism (HP:0004356)](https://hpo.jax.org/app/browse/term/HP:0004356)
- pmid: PMID:25665455
- def: An abnormal result of the the filipin test. In this test is based on the reaction of unesterified cholesterol with fluorescent antibiotic filipin giving a strongly fluorescent, stable cholesterol-filipin complex suitable for in situ detection. An abnormal test result is present if there is a perinuclear accumulation of staining in lysosomal storage organelles (LSOs).
- comment: The filipin test is useful for the evaluation of impaired cholesterol esterification in cultured cells (fibroblasts or others) in Niemann-Pick disease type C.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9212
- synonyms: Positive filipin test(NONE)
